### ISSUE TITLE
Could not import Tempto project into IntelliJ

### ISSUE THREAD

**Issue by sogorkis** [prestodb/tempto#34]
When I try to import project from gradle I get this message:

```
No such property: nexusUsername for class: org.gradle.api.publication.maven.internal.ant.DefaultGroovyMavenDeployer  Consult IDE log for more details (Help | Show Log)
```

It looks like there is a problem in `build.gradle` file.

**Comment by petroav** [comment]
Are you on the latest IntelliJ? I personally haven't imported tempto, just suggesting something to try.

**Comment by sogorkis** [comment]
Fixed: #47
